packages/client/src/miner/pendingBlock.ts

```diff
@@ -107,10 +107,8 @@ export class PendingBlock {
     if (typeof vm.blockchain.getTotalDifficulty !== 'function') {
       throw new Error('cannot get iterator head: blockchain has no getTotalDifficulty function')
     }
```

Comment:
Do we still need this check?

Reply:
I would find it nice if we are still able to at least *run* ethash blockchains, and this check might be necessary for it?

Reply:
You are probably right.  On a related note:  should this still work `npm run client:start:ts -- --dev=pow`?
![image](https://github.com/user-attachments/assets/db3cc7dc-0e42-44d7-a0bb-bd971c9a2c42)

Reply:
No idea about this dev flags